core/client-internal/src/test/java/tachyon/client/file/BaseFileSystemTest.java

```diff
@@ -69,7 +69,6 @@
    */
   @Before
   public void before() {
-    ClientContext.reset();
```

Comment:
I don't see anything in the `before` that modifies the configuration, so it shouldn't be necessary